src/pages/guestbook/index.js

```diff
@@ -27,15 +29,8 @@ class Guestbook extends Component {
 
   componentDidMount() {
     // Gitalk
-    const gitalk = new Gitalk({
-      clientID: '18255f031b5e11edd98a',
-      clientSecret: '2ff6331da9e53f9a91bcc991d38d550c85026714',
-      repo: 'calpa.github.io',
-      owner: 'calpa',
-      admin: ['calpa'],
-      distractionFreeMode: true,
-    });
-    gitalk.render('gitalk-container');
+    const giTalk = new Gitalk(gitalk);
```

Comment:
giTalk -> gitalk

Reply:
@calpa there are two `gitalk` variables in the file one has to be renamed.

Reply:
oh, maybe we can use `GitTalkInstance`?

Reply:
ya I will change to that